src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -227,6 +230,53 @@ void CascadiaSettings::_CreateDefaultProfiles()
 
     _profiles.emplace_back(powershellProfile);
     _profiles.emplace_back(cmdProfile);
+
+    HANDLE readPipe;
+    HANDLE writePipe;
+    SECURITY_ATTRIBUTES sa{ sizeof(sa), nullptr, true };
+    if (CreatePipe(&readPipe, &writePipe, &sa, 0)) {
+        STARTUPINFO si{};
+        si.cb = sizeof(si);
+        si.dwFlags = STARTF_USESTDHANDLES;
+        si.hStdOutput = writePipe;
+        si.hStdError = writePipe;
+
+        PROCESS_INFORMATION pi{};

```

Comment:
Try `wil::unique_process_information` on for size to safely manage the handles that come back in a `PROCESS_INFORMATION` structure.

Reply:
That's great, cleaned up all my handles at the bottom.